### PR TITLE
New leather armor sprites + Refactor

### PR DESCRIPTION
**Pull request by Gyrandola**
## About The Pull Request

- Move the Heavy leather gloves (the ones desert riders use) under the leather subtype.
- Reorganize all leather armor, pants, shoes, gloves in their own files for ease of use.
- Remove some codersprites.
- Adds new leather sprites by **daemoninsideme**, replacing existing ones where able.

image
image
image

### CODE REVIEW

**Comment by Nakalon** [comment]
I'd keep Cynthias sprites, she worked really hard

**Comment by Gyrandola** [comment]
> I'd keep Cynthias sprites, she worked really hard

i had to replace them to keep the leather armor style consistent

**Comment by Nakalon** [comment]
> > I'd keep Cynthias sprites, she worked really hard
> 
> i had to replace them to keep the leather armor style consistent

Ah, I understand, a true shame!

File: code/modules/clothing/rogueclothes/leather/pants.dm [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	desc = "Trousers made of fine leather."
+	icon_state = "leatherpants"
+	max_integrity = 130
+	armor = list("blunt" = 50, "slash" = 25, "stab" = 40, "bullet" = 10, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	prevent_crits = list(BCLASS_CUT, BCLASS_STAB)
+
+/obj/item/clothing/under/roguetown/trou/leather/advanced
+	name = "hardened leather chausses"
+	desc = "Sturdy, durable, flexible. The finest leather to protect your nether regions."
+	icon_state = "leatherpants_masterwork" //no advanced sprite
```
Comment (Nakalon) [inline code/modules/clothing/rogueclothes/leather/pants.dm:13]: So masterowkr and hardened keep the same icon? wouldn't it be better to keep Cynthias on-mob sprites for hardened leather just so theres a difference?

**Review by Nakalon — APPROVED** [review]
Everything seems well, but will wait on a reply to my last comment!

**Pull request merged**